parquet-protobuf/src/main/java/org/apache/parquet/proto/ProtoSchemaConverter.java

```diff
@@ -97,6 +127,46 @@ public MessageType convert(Class<? extends Message> protobufClass) {
 
   private <T> Builder<? extends Builder<?, GroupBuilder<T>>, GroupBuilder<T>> addField(FieldDescriptor descriptor, final GroupBuilder<T> builder) {
     if (descriptor.getJavaType() == JavaType.MESSAGE) {
+      if (unwrapProtoWrappers) {
+        String typeName = descriptor.getMessageType().getFullName();
```

Comment:
is there a reason you are using type equality vs descriptor equality?  (I haven't checked but I would think .Equals would work?  Maybe add a comment if this is intentional.

Reply:
That's a good point. I'll change it to compare the `Descriptor` directly to the Wrapper's Descriptor rather than the name.